template.json

```diff
@@ -1,32 +1,32 @@
 {
-"Actor": {
-  "types": ["character", "crew", "\uD83D\uDD5B clock", "factions"],
-  "character": {
-    "alias": "",
-    "stress": {
+  "Actor": {
```

Comment:
Please make sure to maintain the same amount of tabs

Reply:
Should be fixed.